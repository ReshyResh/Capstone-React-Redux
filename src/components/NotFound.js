import React from 'react'

const NotFound = () => {
    return (
        <div>
            <h1>Page not found!</h1>
        </div>
    )
}

export default NotFound
