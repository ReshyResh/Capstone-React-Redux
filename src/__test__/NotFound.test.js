import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NotFound from '../components/NotFound';

it('404 page renders correctly', () => {
  const tree = renderer
    .create(<Router><NotFound /></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
