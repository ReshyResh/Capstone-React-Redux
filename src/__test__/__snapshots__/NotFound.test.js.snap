// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page renders correctly 1`] = `
<div>
  <h1
    className="white"
    style={
      Object {
        "fontSize": "4em",
        "margin": "auto",
        "textAlign": "center",
      }
    }
  >
     
    Page not found!
    <br />
    Click 
    <a
      href="/"
      onClick={[Function]}
    >
      here 
    </a>
    to go back home! 
  </h1>
</div>
`;
